sonar-plugin-server/src/main/java/jetbrains/buildserver/sonarplugin/buildfeatures/BranchesAndPullRequestsBuildFeature.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright 2000-2020 JetBrains s.r.o.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package jetbrains.buildserver.sonarplugin.buildfeatures;
+
+import java.util.Map;
+
+import org.jetbrains.annotations.NotNull;
+
+import jetbrains.buildServer.serverSide.BuildFeature;
+import jetbrains.buildServer.web.openapi.PluginDescriptor;
+
+/**
+ * Enable the support of SonarQube Branches (https://docs.sonarqube.org/latest/branches/overview/) and Pull-Requests
+ * (https://docs.sonarqube.org/latest/analysis/pull-request/), by providing SONARQUBE_SCANNER_PARAMS environment variable
+ */
+public class BranchesAndPullRequestsBuildFeature extends BuildFeature {
```

Comment:
No dedicated unit test for BuildFeature (main process is in `ParametersProcessor`) ... is there some relevant test todo here ?